### PR TITLE
CPU info: add names

### PR DESCRIPTION
**Pull request by mattwelborn**
## Description
This PR adds display names for cpus.

## Status

- [x] Code base linted
- [x] Ready to go

### CODE REVIEW

File: raw_data/cpu_data/build_cpu_data.py [hunk 1]
```diff
@@ -201,6 +201,18 @@ def parse_date(d):
...
 
+def name(vendor, family, model, clock_speed):
+    if vendor.lower() in family.lower():
+        vendor = ""
+    if family in str(model):
+        family = ""
+    if family.endswith("Processors") and family[:-len("Processors")] in str(model):
+        family = ""
+
+    return f"{vendor} {family} {model} @ {clock_speed/1_000_000_000:.1f} GHz"
```
Comment (dgasmith) [inline raw_data/cpu_data/build_cpu_data.py]: Can you give a few examples of what this looks like for AMD and intel?

Comment (mattwelborn) [reply raw_data/cpu_data/build_cpu_data.py]: Showing some good and some less good results:

```
 Legacy Intel Xeon Processors X5687 @ 3.6 GHz
 Legacy Intel Core Processors i7-3610QM @ 2.3 GHz
 Intel Xeon Processor E7 v3 Family E7-4809V3 @ 2.0 GHz
 Intel Celeron Processor J Series J3060 @ 1.6 GHz
 AMD Opteron™ 6278 @ 2.4 GHz
 AMD PRO A-Series Processors 6th Gen AMD PRO A12-8870 APU @ 3.7 GHz
 AMD Ryzen™ PRO Processors AMD Ryzen™ 7 PRO 2700U @ 2.2 GHz
 AMD Phenom™ 980 @ 3.7 GHz
 8th Generation Intel Core i9 Processors i9-8950HK @ 2.9 GHz
 AMD Ryzen™ 3 1300X @ 3.5 GHz
```

Comment (loriab) [reply raw_data/cpu_data/build_cpu_data.py]: rather needs a "no duplicate words" rule :-)

Comment (mattwelborn) [reply raw_data/cpu_data/build_cpu_data.py]: It's some sort of "no duplicate words or phrases" but only for words/phrases that make sense. The second clause being the hard part.

Comment (dgasmith) [reply raw_data/cpu_data/build_cpu_data.py]: It seems reasonable at the very least?

Comment (mattwelborn) [reply raw_data/cpu_data/build_cpu_data.py]: I guess the word `Processor` is never useful.

**Pull request merged**